packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -733,7 +733,7 @@ describe('plugin-devtools', () => {
             const subpanel = devtools.querySelector('devtools-panel.active mainpane-contents');
             expect(subpanel.textContent).to.equal('insertTextsetRange');
         });
-        it('should select "insertLineBreak"', async () => {
+        it('should select "setRange"', async () => {
```

Comment:
It's `setSelection` now, but I guess you would have noticed it when the test breaks after rebasing :p

Reply:
Now that I think about it, even though `setSelection` sets the *selection* of the VDocument, it is more or less equivalent to "setting the current range of the user edition" so I would not be against renaming it back to `setRange` if someone is emotional about using `setRange`. I mean, the selection *is* the default range that most commands use, so in a way it is not wrong at all. Fuck, now I want to rename it `setRange` and have it act upon `VDocument.selection.range` which would make sense actually... Damn it ! (What do you think ?)

Reply:
Let's just say I'm not particularly emotional about it :-D